src/api-docs.js

```diff
@@ -1625,6 +1678,95 @@ const apiDoc = {
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          }
+        ],
+        requestBody: {
+          description: 'Get the requested proposal by identifier from the atomic swap service.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['proposal_id', 'password'],
+                properties: {
+                  proposal_id: {
+                    description: 'Proposal identifier on the Atomic Swap Service.',
+                    type: 'string',
+                  },
+                  password: {
+                    description: 'Proposal password on the Atomic Swap Service.',
+                    type: 'string',
+                  },
+                }
+              },
+              examples: {
+                fetch: {
+                  summary: 'Successful fetch from the Atomic Swap Service',
+                  value: {
+                    proposal_id: '1a574e6c-7329-4adc-b98c-b70fb20ef919',
+                    password: 'abc123',
+                  },
+                },
+              }
+            }
+          }
+        },
+        responses: {
+          200: {
+            description: 'Fetches proposal data from the service.',
+            content: {
+              'application/json': {
+                examples: {
+                  success: {
+                    summary: 'Success',
+                    value: {
+                      success: {
+                        summary: 'Successful fetch',
+                        value: {
+                          success: true,
+                          proposal: {
+                            proposalId: '1a574e6c-73...',
+                            version: 1,
+                            timestamp: 'Fri Mar 10 2023 23:13:...',
+                            partialTx: 'PartialTx|000100010...',
+                            signatures: null,
+                            history: []
+                          },
+                        },
+                      },
+                    }
+                  },
+                  'service-failure': {
+                    summary: 'Failure on the service side',
+                    value: {
+                      success: {
+                        summary: 'Unsuccessful fetch from the Atomic Swap Service',
+                        value: {
+                          success: false,
+                          error: 'Failure description on the swap service',
+                        },
+                      },
+                    }
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    },
```

Comment:
Missing the register API

Reply:
Fixed it on [595c9a5](https://github.com/HathorNetwork/hathor-wallet-headless/pull/277/commits/595c9a5ffe0cc04e502d7ff1777cd47ba51a4140).